<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>129</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupHeights">
       <property name="maximumSize">
        <size>
         <width>111</width>
         <height>121</height>
        </size>
       </property>
       <property name="title">
        <string>Height range</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_min">
          <property name="text">
           <string>min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderMinHeight">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>91</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
          <property name="tickInterval">
           <number>500</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_max">
          <property name="text">
           <string>max</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderMaxHeight">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>91</width>
            <height>22</height>
           </size>
          </property>
          <property name="minimum">
           <number>200</number>
          </property>
          <property name="maximum">
           <number>4000</number>
          </property>
          <property name="pageStep">
           <number>10</number>
          </property>
          <property name="tracking">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
          <property name="tickInterval">
           <number>500</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupTypeCoord">
       <property name="minimumSize">
        <size>
         <width>111</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>111</width>
         <height>80</height>
        </size>
       </property>
       <property name="title">
        <string>Type coordinate</string>
       </property>
       <widget class="QRadioButton" name="radioButtonGeo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Geo</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonDecart">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Decart</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>109</width>
         <height>48</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
